libs/cdk/src/lib/zone-configuration/event-names.ts

```diff
@@ -68,52 +123,80 @@ export const touchEvents: (keyof WindowEventMap)[] = [
   'touchmove',
   'touchcancel',
 
+  // [DragEvent](https://developer.mozilla.org/en-US/docs/Web/API/DragEvent)
   'drag',
   'dragend',
+  'dragexit',
   'dragenter',
   'dragleave',
   'dragover',
   'dragstart',
   'drop',
 ];
 
-export const formControlsEvents: (keyof WindowEventMap)[] = [
-  'change',
-  'blur',
-  'focus',
-  'contextmenu',
-  'input',
-];
-
 export const globalEvents: (keyof WindowEventMap)[] = [
-  // window
+  'contextmenu',
+  'resize',
   'scroll',
+
+  'abort',
   'load',
+  'loadeddata',
+  'loadedmetadata',
+  'loadstart',
+  'unload',
+
   'error',
 ];
 
-// XHREvent
-export const xhrEvent = ['XHR'];
-
 export const websocketEvents: (keyof WebSocketEventMap)[] = [
   'close',
   'error',
   'message',
   'open',
 ];
 
-export const allEvents: string[] = [
-  ...uiEvent,
-  ...focusEvent,
-  ...selectionEvent,
-  ...mouseEvent,
-  ...wheelEvent,
-  ...inputEvent,
-  ...keyboardEvent,
-  ...compositionEvent,
-  ...touchEvents,
-  ...formControlsEvents,
-  ...globalEvents,
-  ...xhrEvent,
-  ...websocketEvents,
+// XHREvent (group is here to make it easier to target HXR in angular applications. It contains redundant events e.g. 'error')
+export const xhrEvents = ['XHR', 'load', 'error'];
```

Comment:
todo: add convenience method to disable XHR events altogether with XHR flag itself

Reply:
`zoneConfig.rxa.disable.XHR()` maybe?

Where `rxa` is the slot for convenience things?